src/controllers/AuthController.js

```diff
@@ -176,10 +177,39 @@ const resetPassword = async (request, response) => {
   }
 };
 
+const resendMail = async (req, res) => {
+  const { id } = req.params;
+  try {
+    const user = await findUser(id);
+    if (!user) {
+      return responseMessage(res, 404, {
+        error: 'User not found',
+      });
+    }
+
+    const { email, firstName, isVerified } = user;
+    if (isVerified) {
+      return responseMessage(res, 403, {
+        error: 'you\'re already verified',
+      });
+    }
+
+    await verifyUser({ id, email, firstName });
+    return responseMessage(res, 200, {
+      message: 'request sent you\'ll a mail shortly',
```

Comment:
I think there's a typo here. "you'll a mail shortly"

Reply:
done